src/pages/Products/ProductList/ProductList.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import './ProductList.scss';
+import OneProduct from '../OneProduct/OneProduct';
+import '../../../config.js';
```

Comment:
- import 순서 항상 신경써 주세요! 순서는 리팩토링 체크리스트 참고해 주세요!
```suggestion
import React from 'react';
import OneProduct from '../OneProduct/OneProduct';
import '../../../config.js';
import './ProductList.scss';
```